libbeat/docs/shared-autodiscover.asciidoc

```diff
@@ -136,6 +136,75 @@ endif::[]
+endif::[]
+ifeval::["{beatname_lc}"!="filebeat"]
+ disabled by default.
+endif::[]
+`kube_config`:: (Optional) Use given config file as configuration for Kubernetes
+  client. If kube_config is not set, KUBECONFIG environment variable will be
+  checked and if not present it will fall back to InCluster.
+`kube_client_options`:: (Optional) Additional options can be configured for Kubernetes
+  client. Currently client QPS and burst are supported, if not set Kubernetes client's
+  https://pkg.go.dev/k8s.io/client-go/rest#pkg-constants[default QPS and burst] will be used.
+  Example:
+["source","yaml",subs="attributes"]
+-------------------------------------------------------------------------------------
+      kube_client_options:
+        qps: 5
+        burst: 10
+-------------------------------------------------------------------------------------
+
+`resource`:: (Optional) Select the resource to do discovery on. Currently supported
+  Kubernetes resources are `pod`, `service` and `node`. If not configured `resource`
+  defaults to `pod`.
+`scope`:: (Optional) Specify at what level autodiscover needs to be done at. `scope` can
+  either take `node` or `cluster` as values. `node` scope allows discovery of resources in
+  the specified node. `cluster` scope allows cluster wide discovery. Only `pod` and `node` resources
+  can be discovered at node scope.
+`add_resource_metadata`:: (Optional) Specify labels and annotations filters for the extra metadata coming from Node and Namespace.
+ `add_resource_metadata` can be done for `node` or `namespace`. By default all labels will be included
+  while annotations are not added by default. This settings are useful when labels' and annotations'
+  storing requires special handling to avoid overloading the storage output. The enrichment of `node` or `namespace` metadata
+  can be individually disabled by setting `enabled: false`. If resource is `pod` and it is created from a `deployment`, by default
+  the deployment name is added, this can be disabled by set to `false`.
+  Example:
+
+["source","yaml",subs="attributes"]
+-------------------------------------------------------------------------------------
+      add_resource_metadata:
+        namespace:
+          include_labels: ["namespacelabel1"]
+        node:
+          include_labels: ["nodelabel2"]
+          include_annotations: ["nodeannotation1"]
+        deployment: false
+-------------------------------------------------------------------------------------
+
+`unique`:: (Optional) Defaults to `false`. Marking an autodiscover provider as unique results into
+  making the provider to enable the provided templates only when it will gain the leader lease.
+  This setting can only be combined with `cluster` scope. When `unique` is enabled enabled, `resource`
+  and `add_resource_metadata` settings are not taken into account.
+`leader_lease`:: (Optional) Defaults to `{beatname_lc}-cluster-leader`. This will be name of the lock lease.
```

Comment:
```suggestion
`leader_lease`:: (Optional) Defaults to +{beatname_lc}-cluster-leader+. This will be name of the lock lease.
```

Reply:
Found this existing nit in the content you moved. This change is needed so that {beatname_lc} resolves correctly.

Reply:
I do not undestand this templating system, so I accept your suggestion.

Reply:
Well, if I read the asciidoc reference well, the plus sign disables formating inside + symbols.
Here you replace emphasize with plus symbols which has totaly different function. I see you want to keep  {beatname_lc} unresolved at the moment.

Reply:
I see similar pattern used on line 8 of the file altought I do not understand how it complies with asciidocs standard.
Either way this bug was there even before.

Reply:
I am sorry, I read wrong documentation on asciidoctor, now I have fount the correct one.
https://asciidoc-py.github.io/userguide.html#X81